internal/exec/resolvable/resolvable.go

```diff
@@ -380,13 +384,46 @@ func findMethod(t reflect.Type, name string) int {
 	return -1
 }
 
-func findField(t reflect.Type, name string) int {
+func findField(t reflect.Type, name string, index []int) []int {
 	for i := 0; i < t.NumField(); i++ {
-		if strings.EqualFold(stripUnderscore(name), stripUnderscore(t.Field(i).Name)) {
-			return i
+		field := t.Field(i)
+
+		if field.Type.Kind() == reflect.Struct && field.Anonymous {
+			newIndex := findField(field.Type, name, []int{i})
+			if len(newIndex) > 1 {
+				return append(index, newIndex...)
+			}
+		}
+
+		if strings.EqualFold(stripUnderscore(name), stripUnderscore(field.Name)) {
+			return append(index, i)
 		}
 	}
-	return -1
+
+	return index
+}
+
+// getFieldCount helps resolve ambiguity when more than one embedded struct contains fields with the same name.
```

Comment:
When this ambiguity happens do you throw some kind of an error? I'd appreciate a unit tests that catches this case.

Reply:
Added unit test [here](https://github.com/graph-gophers/graphql-go/pull/338/files#diff-d7906d668830f592334a2a80cf6d298cR3030).
Also removed `get` prefix.